controllers/operands/operandHandler.go

```diff
@@ -148,6 +144,34 @@ func (h *OperandHandler) addOperands(scheme *runtime.Scheme, hc *hcov1beta1.Hype
 	}
 }
 
+type GetHandler func(log.Logger, client.Client, *runtime.Scheme, *hcov1beta1.HyperConverged) (Operand, error)
+
+func (h *OperandHandler) addOperand(scheme *runtime.Scheme, hc *hcov1beta1.HyperConverged, getHandler GetHandler) {
+	handler, err := getHandler(logger, h.client, scheme, hc)
+	if err != nil {
+		logger.Error(err, "can't create ConsoleQuickStarts objects")
```

Comment:
why `ConsoleQuickStarts` specifically in this generic function?

Reply:
history. will fix